.github/ISSUE_TEMPLATE/mapping_issue_report.yml

```diff
@@ -0,0 +1,47 @@
+name: Mapping issue report
+description: Report a mapping problem you found while playing.
+title: ''
+labels: 'Group: Mapping'
+body:
+
+- type: checkboxes
+  attributes:
+    label: There is no other issue reporting the same problem.
+    description: Please take some time and try to find another issue reporting the same problem you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+
+-type: dropdown
+  attributes:
+    label: Map name
+    description: What map did you encounter the problem on?
+    options:
+    - BoxStation
+    - PogStation
+    - FallStation
+    - MiniStation
+    - TestStation
+    - SquareStation
+    - UnrealStation
+    - OutpostStation
+    - Lavaland
+    - Asteroid
+  validations:
```

Comment:
aren't we going to have "another" option?

Reply:
sure, I can add that. Although it would be preferred to update the template if we add more scenes in the future.

Reply:
done